web/src/routes/SurveyPages/SurveyListPage.tsx

```diff
@@ -291,8 +224,10 @@ export default function SurveyListPage() {
                 </Title>
                 <Authlist theme={theme}>
```

Comment:
Please modify this to `Auth'L'ist`. See [this code](https://github.com/kookmin-sw/capstone-2023-40/blob/9f1d23cb17ee1673bfba2a1f3cb548504f53450e/web/src/routes/SurveyPages/SurveyListPage.tsx#L71)